Tests/js/views/tabs/assets/ez-locationviewdetailstabview-tests.js

```diff
@@ -0,0 +1,287 @@
+                method: 'get',
+                args: [Mock.Value.String],
+                run: function (attr) {
+                    if (attr === 'currentVersion') {
+                        return that.versionMock;
+                    } else if (attr === 'resources') {
+                        return {Owner: that.owner};
+                    } else {
+                        Y.fail("Unexpected parameter " + attr + " for content mock");
+                    }
+                }
+            });
+
+            Mock.expect(that.versionMock, {
+                method: 'get',
+                args: ['resources'],
+                returns: {Creator: this.creator}
+            });
+
+            this.view = new Y.eZ.LocationViewDetailsTabView({
+                content: that.contentMock
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should fire loadUser Event with different owner and creator": function () {
+            var that = this,
+                ownerCalled = false,
+                creatorCalled = false;
+
+            this.view.once('loadUser', function (e) {
+                ownerCalled = true;
+                Assert.areSame(
+                    that.creator,
+                    e.userId,
+                    "The event facade should contain the creator id"
+                );
+
+                that.view.once('loadUser', function (e) {
+                    creatorCalled = true;
+                    Assert.areSame(
+                        that.owner,
+                        e.userId,
+                        "The event facade should contain the owner id"
+                    );
+                });
+            });
+
+            this.view.set('active', true);
+
+            Assert.isTrue(ownerCalled, "loadUser should have been called for the owner");
+            Assert.isTrue(creatorCalled, "loadUser should have been called for the creator");
+
+        },
+
+        "Should fire loadUser Event with same owner and creator": function () {
+            var that = this,
+                ownerCalled = false,
+                creatorCalled = false;
+            this.owner = "13";
+            this.view.render = function () {};
+
+            this.view.once('loadUser', function (e) {
+                ownerCalled = true;
+                Assert.areSame(
+                    that.creator,
+                    e.userId,
+                    "The event facade should contain the creator id"
+                );
+
+                that.view.once('loadUser', function (e) {
+                    creatorCalled = true;
+                });
+            });
+
+            this.view.set('active', true);
+
+            Assert.isTrue(ownerCalled, "loadUser should have been called for the owner");
+            Assert.isFalse(creatorCalled, "loadUser should not be called for the creator");
+
+            this.view.set('creator', 'Paul Mc Cartney');
```

Comment:
you should test that in different function and put an object in the creator attribute.